docs/pbc/all/dynamic-multistore/202404.0/tutorials-and-howtos/how-to-configure-application-timezone.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Configure application context.
+description: This guide shows how to configure timezone for store application
+last_updated: Jun 5, 2024
+template: howto-guide-template
+---
+
+This document describes how to create and configure store context application. 
+
+## Create and configure application timezone in database
+
+To setup timezone for each application in store need add configuration in `spy_store_context` table. 
+
+The table contains the configuration of  and has the following columns:
+
+| COLUMN | SPECIFICATION |
+| --- | --- |
+| id_store_context | ID of the store context configuration. |
+| application_context_collection | A JSON-formatted string containing the configuration for each application in the store. |
+| fk_store | Foreign key of the store ID.  |
+
+The following example shows a possible value of the `spy_store_context.application_context_collection` field configured for the some store with default timezone `Europe/Berlin` and different timezone for `zed` and `yves` applications. 
```

Comment:
Timezone is already configured value that does not show customers how to use store context, please provide another example, e.g. with Date Format. (Can be done in one document, [how-to-use-store-timezone.md](https://github.com/spryker/spryker-docs/pull/2693/files#diff-9bd7ea291f34f92d0c65dccd83f5a9c42f6edc2bddc53fc6bf82c7d8c004affa) can be removed)

Reply:
So, will be delete it?

Reply:
Discussed